src/zeroband/inference/config.py

```diff
@@ -280,6 +281,9 @@ class Config(BaseSettings):
     # The RL configuration. If None, inference will run in a non-RL setting.
     rl: Annotated[RLConfig | None, Field(default=None)]
 
+    # The elastic reasoning configuration
+    elastic_reasoning: Annotated[ElasticReasoningConfig, Field(default=ElasticReasoningConfig())]
```

Comment:
```suggestion
    elastic_reasoning: ElasticReasoningConfig,  =  ElasticReasoningConfig() 
 
```

isn't better to keep like this ?